scripts/data.js

```diff
@@ -0,0 +1,237 @@
+    difficulty: ' Super Hard',
+    expectedtime: ' 60 minutes',
+    answer: 'N/A',
+  },
+};
+
+const htmlbody = document.querySelector('.htmlbody');
+const cc = document.createElement('section');
+cc.id = 'coding-challenges';
+cc.className = 'coding-challenges';
+
+function dataLoader() {
+  const title = new Array(7);
+  const question = new Array(7);
+  const difficulty = new Array(7);
+  const expectedtime = new Array(7);
+  Object.values(challenges).forEach((val, i) => {
+    title[i] = val.title;
+    question[i] = val.question;
+    difficulty[i] = val.difficulty;
+    expectedtime[i] = val.expectedtime;
+  });
+  cc.innerHTML = `
+                  <h1 class="coding-header mt-5 d-flex justify-content-center">Featured Coding Challenges</h1>
+                  <div class="d-flex flex-column flex-wrap cc">
+                    <div class="challenge one mx-2 mt-4">
+                        <h2>${title[0]}</h2>
+                        <span><mark>${question[0]}</mark></span><br>
+                        <label class="fw-bold">Difficulty: </label><span>${difficulty[0]}</span><br>
+                        <label class="fw-bold">Expected Completion Time: </label><span>${expectedtime[0]}</span><br>
+                        <button type="button" class="sub oneb btn mt-4">Start Challenge</button>
+                    </div>
+                    <div class="challenge two mx-2 mt-4">
+                        <h2>${title[1]}</h2>
+                        <span><mark>${question[1]}</mark></span><br>
+                        <label class="fw-bold">Difficulty: </label><span>${difficulty[1]}</span><br>
+                        <label class="fw-bold">Expected Completion Time: </label><span>${expectedtime[1]}</span><br>
+                        <button type="button" class="sub twob btn mt-4">Start Challenge</button>
+                    </div>
+                    <div class="challenge three mx-2 mt-4">
+                        <h2>${title[2]}</h2>
+                        <span><mark>${question[2]}</mark></span><br>
+                        <label class="fw-bold">Difficulty: </label><span>${difficulty[2]}</span><br>
+                        <label class="fw-bold">Expected Completion Time: </label><span>${expectedtime[2]}</span><br>
+                        <button type="button" class="sub threeb btn mt-4">Start Challenge</button>
+                    </div>
+                    <div class="challenge four mx-2 mt-4">
+                        <h2>${title[3]}</h2>
+                        <span><mark>${question[3]}</mark></span><br>
+                        <label class="fw-bold">Difficulty: </label><span>${difficulty[3]}</span><br>
+                        <label class="fw-bold">Expected Completion Time: </label><span>${expectedtime[3]}</span><br>
+                        <button type="button" class="sub fourb btn mt-4">Start Challenge</button>
+                    </div>
```

Comment:
### Your Section
![flex-wrap](https://user-images.githubusercontent.com/93039359/181395159-b3b4bd12-914b-432e-b68f-58f4c79654ef.png)

### Design Section
![speaker-section](https://user-images.githubusercontent.com/93039359/181395283-2f48bb2e-c750-452f-88da-fc007ff781c2.png)

- [ ] Please make your section look like the design. You can see it is going to the right but you can use `flex-wrap` or **grid** instead of **flex** to fix this issue :blush:

Reply:
- [ ] [REQUIRED] Good job on implementing changes requested by previous reviewers but you need to arrange this section cause it is very important to have the same alignment as the original design to give to the user a great user experience :+1: 
check the screenshots below for two different screen size :arrow_down: 

![Screenshot from 2022-07-28 10-10-56](https://user-images.githubusercontent.com/6615394/181486824-d4c67a1f-3468-47fb-8269-5eab8d80d01f.png)

![Screenshot from 2022-07-28 10-11-57](https://user-images.githubusercontent.com/6615394/181486819-1782df42-1f47-4671-93af-cf896bd95d57.png)